docs/admin/authentication.md

```diff
@@ -580,6 +580,53 @@ Starting in 1.6, the ABAC and RBAC authorizers require explicit authorization of
 `system:anonymous` user or the `system:unauthenticated` group, so legacy policy rules
 that grant access to the `*` user or `*` group do not include anonymous users.
 
+## User impersonation
+
+A user can act as another user through impersonation headers, which let requests
+manually override the user info they authenticate as. For example an admin could
```

Comment:
It also might be good to give practical uses:

1. Simple two-tier permissions... a user with low day-to-day permissions that can act as a powerful user as needed to avoid fat-fingers:
   ```sh
   $ kubectl drain mynode
   Error from server (Forbidden): User "clark" cannot get nodes at the cluster scope. (get nodes mynode)

   $ kubectl drain mynode --as=superman
   node "mynode" cordoned
   node "mynode" drained
   ```

2. A trusted component with full impersonation powers and information about users (username, groups, extra attributes) can make API calls on their behalf using impersonation, limited to their permissions.